src/main/java/jhoselineteran/assigment2/TotalPoints.java

```diff
@@ -0,0 +1,67 @@
+package jhoselineteran.assigment2;
+
+import java.util.logging.Logger;
+
+/**
+ * The TotalPoints class provides a method to
+ * calculate the total points earned by a team in a series of games.
+ */
+public final class TotalPoints {
+  /**
+   * Logger instance for logging information.
+   */
+  private static final Logger LOGGER =
+          Logger.getLogger(TotalPoints.class.getName());
+  /**
+   * Constant representing the points awarded for a win in a game.
+   */
+
+  private static final int WIN_POINTS = 3;
+
+  private TotalPoints() {
+    // Constructor privado para ocultar el constructor predeterminado
```

Comment:
Please don't add anything in Spanish. Fix the indentation level in your classes.